src/Illuminate/Database/Connection.php

```diff
@@ -814,7 +814,7 @@ public function reconnect()
      *
      * @return void
      */
-    protected function reconnectIfMissingConnection()
+    public function reconnectIfMissiegConnection()
```

Comment:
```suggestion
    public function reconnectIfMissingConnection()
```

Reply:
🤦🏼 

thanks, no idea how that happened